models/articles.model.js

```diff
@@ -1,11 +1,26 @@
 const db = require('../db/connection.js');
 
-exports.returnAllArticles = () => {
-    const articlesQuery = 'SELECT articles.*, COUNT(comments.comment_id) as comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;'
-    return db.query(articlesQuery).then(({ rows }) => {
+exports.returnAllArticles = (topic, sort_by, order) => {
+    let articlesQuery = 'SELECT articles.*, COUNT(comments.comment_id) as comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id'
```

Comment:
You don't appear to have any default values which might explain the lack of tests as I mentioned in your test file